components/event-sources/reconciler/httpsource/controller_test.go

```diff
@@ -58,15 +59,16 @@ func TestNewController(t *testing.T) {
 		NewConfigMap("", logging.ConfigMapName()),
 	)
 	ctx, informers := rt.SetupFakeContext(t)
+	fakeAuthClientset := fakeauthclient.NewSimpleClientset()
 
 	// expected informers: HTTPSource, Channel, Knative Service
+	// AuthV1 informer doesn't have Knative injection informers
+	ctrler := newController(ctx, cmw, fakeAuthClientset)
```

Comment:
This code made more sense where it used to be. First we check that the injection worked as expected, and only _then_ we create the controller. Mixing unrelated actions makes the test confusing IMO.

Reply:
Not needed anymore.